### ISSUE TITLE
increase aarch64 reserved partition size

### ISSUE THREAD

**Issue by M1cha** [coreos/fedora-coreos-tracker#1353]
### Describe the enhancement

So first off: There's no documentation for the aarch64 partition table. I'm assuming partition 1 is meant for firmwares and bootloaders.

Also, the numbers below assume 512Byte sectors.

The EFI partition currently starts at sector 4096 which is way too low for rk3399's uboot image. RK3399 distros usually reserve 32768 sectors because a typical uboot binary is more than 9MB in size.
That means that I currently can't use coreos-installer because it gives me no way of installing the bootloader without overwriting the EFI partition.

### System details

- Bare Metal Rock Pi 4

**Comment by dustymabe** [comment]
Yeah. Much more information on this over in https://github.com/coreos/fedora-coreos-tracker/issues/855. See where this is implemented in [coreos-assembler](https://github.com/coreos/coreos-assembler/blob/538402ec655961f7a79e9745c9a3af67e1123e39/src/create_disk.sh#L151-L158).

We should probably add some information to https://docs.fedoraproject.org/en-US/fedora-coreos/storage/#_disk_layout

**Comment by M1cha** [comment]
Maybe we should change the install process for aarch64 so it is aware of the partitions and copies partitions rather than a full disk image? this would allow providing the size of the reserved partition on the CLI. For x64 this might also be useful since it would simplify the process of saving partitions.
